### PR TITLE
Update changelog to reflect PR 5792 and 5796.

### PR DESCRIPTION
**Pull request by brettle**
Forgot to include these updates in those PRs.

### CODE REVIEW

**Review by ad-daniel — DISMISSED** [review]
Indeed, thank you

File: docs/reference/changelog-r2023.md [hunk 1]
```diff
@@ -28,8 +28,9 @@ Released on ??
...
     - Fixed crashes resulting from updating a DEF node whose USE node is contained in a PROTO field triggering the regeneration ([#5676](https://github.com/cyberbotics/webots/pull/5676)).
     - Fixed crashes resulting from copy or DEF/USE update in [`Solid.boundingObject`](solid.md) field ([#5686](https://github.com/cyberbotics/webots/pull/5686)).
-    - Fixed silent ignoring of more that 10 contact points when detecting collisions. The limit has been raised to 100 contact points and a warning is now logged if that limit is reached ([#5776](https://github.com/cyberbotics/webots/pull/5776)).
+    - Fixed silent ignoring of more that 10 contact points when detecting collisions. All contact points are now found and the deepest 10 are used. ([#5792](https://github.com/cyberbotics/webots/pull/5792)).
     - Fixed performance issues when retrieving multiple times a node's field defined in a PROTO body using the [Supervisor API](supervisor.md) ([#5774](https://github.com/cyberbotics/webots/pull/5774)).
+    - Fixed thread-safety issue when robots collide. ([#5796](https://github.com/cyberbotics/webots/pull/5796))
```
Comment (stefaniapedrazzi) [inline docs/reference/changelog-r2023.md:33]: I would also specify that this error could occur only if physics multi-threading is enabled (note that this is disabled by default and not many users are enabling it)
```suggestion
    - Fixed thread-safety issue when robots collide ([#5796](https://github.com/cyberbotics/webots/pull/5796)).
```

**Review by stefaniapedrazzi — CHANGES_REQUESTED** [review]

File: docs/reference/changelog-r2023.md [hunk 2]
```diff
@@ -28,8 +28,9 @@ Released on ??
...
     - Fixed crashes resulting from updating a DEF node whose USE node is contained in a PROTO field triggering the regeneration ([#5676](https://github.com/cyberbotics/webots/pull/5676)).
     - Fixed crashes resulting from copy or DEF/USE update in [`Solid.boundingObject`](solid.md) field ([#5686](https://github.com/cyberbotics/webots/pull/5686)).
-    - Fixed silent ignoring of more that 10 contact points when detecting collisions. The limit has been raised to 100 contact points and a warning is now logged if that limit is reached ([#5776](https://github.com/cyberbotics/webots/pull/5776)).
+    - Fixed silent ignoring of more that 10 contact points when detecting collisions. All contact points are now found and the deepest 10 are used. ([#5792](https://github.com/cyberbotics/webots/pull/5792)).
     - Fixed performance issues when retrieving multiple times a node's field defined in a PROTO body using the [Supervisor API](supervisor.md) ([#5774](https://github.com/cyberbotics/webots/pull/5774)).
+    - Fixed thread-safety issue when robots collide ([#5796](https://github.com/cyberbotics/webots/pull/5796)).
```
Comment (brettle) [inline docs/reference/changelog-r2023.md:33]:
```suggestion
    - Fixed thread-safety issue when robots collide. This was only an issue if physics multi-threading was enabled (which is not the default and rarely used) ([#5796](https://github.com/cyberbotics/webots/pull/5796)).
```

**Review by stefaniapedrazzi — APPROVED** [review]
Thank you!

**Pull request merged**